# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../small/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

int i is redefined ...

  - Rene Rebe <rene@exactcode.de>

--- ./amx/amx.c.vanilla	2005-04-11 10:53:41.000000000 +0200
+++ ./amx/amx.c	2005-04-11 10:53:46.000000000 +0200
@@ -957,7 +957,7 @@
    */
   #if BYTE_ORDER==BIG_ENDIAN
     AMX_FUNCSTUB *fs;
-    int i,num;
+    int num;
 
     fs=GETENTRY(hdr,natives,0);
     num=NUMENTRIES(hdr,natives,libraries);
